src/main/java/net/dv8tion/jda/api/interactions/callbacks/IMessageEditCallback.java

```diff
@@ -130,9 +130,14 @@ default MessageEditCallbackAction editMessage(@Nonnull String content)
      *         The new message components, such as {@link ActionRow}
      *
      * @throws IllegalArgumentException
-     *         If the provided components are null
+     *         <ul>
+     *             <li>If any of the provided LayoutComponents is null</li>
+     *             <li>If any of the provided LayoutComponents are not compatible with messages</li>
```

Comment:
This also applies to components *inside* the layouts so lets abstract it.
```suggestion
     *             <li>If any of the provided Components are not compatible with messages</li>
```